include/experimental/__p1673_bits/blas1_givens.hpp

```diff
@@ -190,7 +191,7 @@ void givens_rotation_setup(const Real f,
 }
 
 namespace impl {
-template<class Real>
+template<std::floating_point Real>
```

Comment:
This change is harmless, but [technically unnecessary](https://eel.is/c++draft/complex.numbers.general); instantiating `std::complex<T>` with `T` other than `float`, `double`, or `long double` has unspecified effect.  It's totally OK to leave this change, though, as it could make error messages nicer on some implementations.